docs/get-started/getting-started-gcp.md

```diff
@@ -89,8 +93,11 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample 
-[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years 
-[Mortgage data](https://docs.rapids.ai/datasets/mortgage-data).
+[Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb).
+
+To get the input data of the sample notebook, we need to download the full 17 years' [Mortgage 
+data](https://docs.rapids.ai/datasets/mortgage-data) as `mortgage_2000-2016.tgz`, uncompress and 
+upload it to a GCS bucket. Above notebook has handy commands to do this step on master node.
```

Comment:
do we want to put anything about how to upload?  perhaps just a link to gcs docs

Reply:
@tgravescs The above notebook has detailed steps on how to download, uncompress and upload. So I thought we do not duplicate it in the doc here?